Hiyobot/cogs/general/info.py

```diff
@@ -35,17 +30,17 @@ async def _botinfo(self, ctx: commands.Context):
         message_latency2 = time.perf_counter()
```

Comment:
You need to edit the code below.

```suggestion
        latency = round((time.perf_counter() - message_latency1) * 1000, 2
```